### ISSUE TITLE
A question

### ISSUE THREAD

**Issue by cugsunjie** [KonstantinosF/Classification-of-Hyperspectral-Image#2]
Thanks for the work, I have tried the code, but it  can not reach a high accuracy result as posted in the original paper.
![image](https://user-images.githubusercontent.com/40532078/47616845-ee559500-dafc-11e8-8c0f-9281cf786930.png)
The following are my results:
![image](https://user-images.githubusercontent.com/40532078/47616811-8e5eee80-dafc-11e8-9c5f-84edc5df7b78.png)
Train on 29685 samples, validate on 2563 samples
Epoch 1/15
29685/29685 [==============================] - 8s 258us/step - loss: 2.5315 - acc: 0.2048 - val_loss: 2.4267 - val_acc: 0.3531
Epoch 2/15
29685/29685 [==============================] - 7s 239us/step - loss: 1.9284 - acc: 0.4615 - val_loss: 1.8825 - val_acc: 0.5099
Epoch 3/15
29685/29685 [==============================] - 7s 240us/step - loss: 1.4118 - acc: 0.5917 - val_loss: 1.3746 - val_acc: 0.6278
Epoch 4/15
29685/29685 [==============================] - 7s 239us/step - loss: 1.0785 - acc: 0.6655 - val_loss: 1.1172 - val_acc: 0.6859
Epoch 5/15
29685/29685 [==============================] - 7s 240us/step - loss: 0.8887 - acc: 0.7122 - val_loss: 0.9545 - val_acc: 0.7323
Epoch 6/15
29685/29685 [==============================] - 7s 240us/step - loss: 0.7718 - acc: 0.7483 - val_loss: 0.8611 - val_acc: 0.7554
Epoch 7/15
29685/29685 [==============================] - 7s 240us/step - loss: 0.6798 - acc: 0.7747 - val_loss: 0.7822 - val_acc: 0.7706
Epoch 8/15
29685/29685 [==============================] - 7s 241us/step - loss: 0.6123 - acc: 0.7953 - val_loss: 0.7271 - val_acc: 0.7799
Epoch 9/15
29685/29685 [==============================] - 7s 240us/step - loss: 0.5648 - acc: 0.8120 - val_loss: 0.6752 - val_acc: 0.7936
Epoch 10/15
29685/29685 [==============================] - 7s 240us/step - loss: 0.5189 - acc: 0.8262 - val_loss: 0.6337 - val_acc: 0.8053
Epoch 11/15
29685/29685 [==============================] - 7s 241us/step - loss: 0.4804 - acc: 0.8420 - val_loss: 0.5988 - val_acc: 0.8092
Epoch 12/15
29685/29685 [==============================] - 7s 240us/step - loss: 0.4488 - acc: 0.8496 - val_loss: 0.5717 - val_acc: 0.8174
Epoch 13/15
29685/29685 [==============================] - 7s 242us/step - loss: 0.4181 - acc: 0.8618 - val_loss: 0.5408 - val_acc: 0.8205
Epoch 14/15
29685/29685 [==============================] - 7s 243us/step - loss: 0.3959 - acc: 0.8701 - val_loss: 0.5088 - val_acc: 0.8283
Epoch 15/15
29685/29685 [==============================] - 7s 241us/step - loss: 0.3730 - acc: 0.8765 - val_loss: 0.5010 - val_acc: 0.8221
Can you tell me what is the reason? Thanks in advance.

**Comment by KonstantinosF** [comment]
Thank you for your message.

The paper that I am mentioning at the description is an independent work.
It does not correlate with my code. Thus the results are not coming from my
code.

I mention the paper, just to illustrate the reasoning that I followed to
build my project.

Hope that clarifies.

Best regards

Στις Κυρ, 28 Οκτ 2018 στις 3:01 μ.μ., ο/η cugsunjie <
notifications@github.com> έγραψε:

**Comment by cheer00** [comment]
@cugsunjie @KonstantinosF 
Hello, I have some trouble when running this code, could you please tell me the version of the used python, keras, tensorflow or theano?
 Thanks very much.

**Comment by KonstantinosF** [comment]
Hello,

Thank you for your message.

You should definitely python 3. If you install keras with "conda install
keras" then it shouldn't be a problem. Maybe you can provide me with more
informations about your problem.

Στις Σάβ, 10 Νοε 2018 στις 6:54 π.μ., ο/η cheer00 <notifications@github.com>
έγραψε:
